src/main/java/origami_editor/editor/folded_figure/FoldedFigure.java

```diff
@@ -275,138 +275,48 @@ void folding_estimation_camera_configure(Camera creasePatternCamera, LineSegment
-            folding_estimated_01(lineSegmentSet);
-            estimationStep = EstimationStep.STEP_1;
-            displayStyle = DisplayStyle.DEVELOPMENT_1;
-            folding_estimated_02();
-            estimationStep = EstimationStep.STEP_2;
-            displayStyle = DisplayStyle.WIRE_2;
-        } else if ((estimationStep == EstimationStep.STEP_0) && (estimationOrder == EstimationOrder.ORDER_3)) {
-            estimated_initialize();
-            folding_estimated_01(lineSegmentSet);
-            estimationStep = EstimationStep.STEP_1;
-            displayStyle = DisplayStyle.DEVELOPMENT_1;
-            folding_estimated_02();
-            estimationStep = EstimationStep.STEP_2;
-            displayStyle = DisplayStyle.WIRE_2;
-            folding_estimated_03();
-            estimationStep = EstimationStep.STEP_3;
-            displayStyle = DisplayStyle.TRANSPARENT_3;
-        } else if ((estimationStep == EstimationStep.STEP_0) && (estimationOrder == EstimationOrder.ORDER_5)) {
-            estimated_initialize();
-            folding_estimated_01(lineSegmentSet);
-            estimationStep = EstimationStep.STEP_1;
-            displayStyle = DisplayStyle.DEVELOPMENT_1;
-            folding_estimated_02();
-            estimationStep = EstimationStep.STEP_2;
-            displayStyle = DisplayStyle.WIRE_2;
-            folding_estimated_03();
-            estimationStep = EstimationStep.STEP_3;
-            displayStyle = DisplayStyle.TRANSPARENT_3;
-            folding_estimated_04();
-            estimationStep = EstimationStep.STEP_4;
-            displayStyle = DisplayStyle.DEVELOPMENT_4;
-            folding_estimated_05();
-            estimationStep = EstimationStep.STEP_5;
-            displayStyle = DisplayStyle.PAPER_5;
-            if ((discovered_fold_cases == 0) && (!findAnotherOverlapValid)) {
-                estimationStep = EstimationStep.STEP_3;
-                displayStyle = DisplayStyle.TRANSPARENT_3;
-            }
-        } else if ((estimationStep == EstimationStep.STEP_1) && (estimationOrder == EstimationOrder.ORDER_1)) {
-        } else if ((estimationStep == EstimationStep.STEP_1) && (estimationOrder == EstimationOrder.ORDER_2)) {
-            folding_estimated_02();
-            estimationStep = EstimationStep.STEP_2;
-            displayStyle = DisplayStyle.WIRE_2;
-        } else if ((estimationStep == EstimationStep.STEP_1) && (estimationOrder == EstimationOrder.ORDER_3)) {
-            folding_estimated_02();
-            estimationStep = EstimationStep.STEP_2;
-            displayStyle = DisplayStyle.WIRE_2;
-            folding_estimated_03();
-            estimationStep = EstimationStep.STEP_3;
-            displayStyle = DisplayStyle.TRANSPARENT_3;
-        } else if ((estimationStep == EstimationStep.STEP_1) && (estimationOrder == EstimationOrder.ORDER_5)) {
+        }
+        if (estimationStep == EstimationStep.STEP_1 && order.isAtLeast(EstimationOrder.ORDER_2)) {
```

Comment:
Nice!

Reply:
Yeah, makes a lot more sense doesn't it?